code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm

```diff
@@ -227,3 +227,12 @@
 	)
 	result = /obj/item/food/meatclown
 	subcategory = CAT_MEAT
+
+/datum/crafting_recipe/food/friedchicken
+	name = "Fried Chicken"
+	reqs = list(
+		/obj/item/food/meat/slab/chicken = 1,
+		/datum/reagent/consumable/cooking_oil = 10
```

Comment:
some absolute legend is going to extract cooking oil from meat, get some chicken meat, and craft fried chicken without a deep fryer (or even a source of heat)

I, for one, think this is cool and should be left in